helm/examples/efs/hpcc-efs/templates/dalistorage-sc.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: {{ printf "dali-%s-sc" (include "hpcc-efs.fullname" $) }}
+
+provisioner: aws.io/aws-efs
+#provisioner: example.com/aws-efs
+
+reclaimPolicy: Retain
+# The reclaimPolicy "Delete" is the default but doesn't work for EFS. Also there is no "Recycle"
```

Comment:
We set reclaimPolicy in the other persist storage example to Retain also, because the example are intentionally persisting data. 
I don't think this comment helps clarify, I'd remove it.